dht/dht.go

```diff
@@ -69,7 +70,8 @@ type distributedHashTable struct {
 	addrsSorted   []wire.Address
 
 	addrsBySignatoryMu *sync.Mutex
-	addrsBySignatory   map[id.Signatory]wire.Address
+	addrsBySignatory   *hbm.Map
+	// addrsBySignatory   map[id.Signatory]wire.Address
```

Comment:
Instead of introducing a new dependency, it would be preferable to just have two maps: `addrsBySignatory` and `signatoriesByAddr` that are both protected by the same mutex, and updated in tandem. Other than `go-ethereum`, Airwave currently has very slim dependencies and we are trying to keep them slim (we are even hoping to remove `go-ethereum` eventually).

Reply:
That sounds reasonable, I'll refactor it (although, the introduced dependecy is a hard fork of the original repository frozen at a particular version).